client/routes.go

```diff
@@ -0,0 +1,14 @@
+package client
+
+import (
+	"github.com/gorilla/mux"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+)
+
+// Register routes
+func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router, cdc *codec.Codec) {
+	RegisterRPCRoutes(cliCtx, r)
```

Comment:
Once we remove the need for a codec as an argument, this should really only be `RegisterRoutes` 👍

Reply:
I'm not sure that will be possible given that client doesn't define it's own "client" level cdc